app/auth/views.py

```diff
@@ -74,7 +76,8 @@ def login():
     :return: Login page.
     """
     current_app.logger.info('Start function login() [VIEW]')
-
+    global still_locked
+    global time_to_wait
```

Comment:
Time_to_wait should be a User attribute, because now it would be shared with everyone running the server.